### PR TITLE
make reflow first line configurable

### PR DESCRIPTION
**Pull request by glensc**
reflow method is nice, except if first line is cut, it changes behaviour. 

so i rather leave first line as is, but still reflow rest of the lines. the default behaviour is the current one.

### CODE REVIEW

File: agito.py [hunk 1]
```diff
@@ -1026,6 +1031,10 @@ def check_svn_repo():
 merge_callbacks = config.get('MERGE_CALLBACKS', {}).copy()
 merge_callbacks['svn:mergeinfo'] = mergeinfo_callback
 
+merge_callbacks = config.get('MERGE_CALLBACKS', {}).copy()
+# Whether to reflow first line in reflow_text
+REFLOW_FIRST_LINE = config.get('REFLOW_FIRST_LINE', True)
```
Comment (fragglet) [inline agito.py]: Please change this to be lowercase, since it isn't really a constant.

File: agito.py [hunk 2]
```diff
@@ -1026,6 +1031,10 @@ def check_svn_repo():
 merge_callbacks = config.get('MERGE_CALLBACKS', {}).copy()
 merge_callbacks['svn:mergeinfo'] = mergeinfo_callback
 
+merge_callbacks = config.get('MERGE_CALLBACKS', {}).copy()
```
Comment (fragglet) [inline agito.py]: Is this duplicated line a mistake?

File: example.cfg [hunk 3]
```diff
@@ -86,6 +86,9 @@ COMMIT_MESSAGE_FILTERS = [
 	agito.append_branch_info,  # Append SVN branch/rev info.
 ]
 
+# Configuration for agito.append_branch_info
```
Comment (fragglet) [inline example.cfg]: Please provide more detail about what this option does, since this example config file serves as a kind of defacto documentation.

**Comment by glensc** [comment]
made the changes, but i'm not sure is the option description better now?

**Comment by fragglet** [comment]
Looks good, thanks!

**Pull request merged**